src/components/FloatingHelper/content/HelperContent.protractor.driver.ts

```diff
@@ -0,0 +1,18 @@
+import {ElementFinder} from 'protractor';
+import {DataHooks} from './DataHooks';
+import {textDriverFactory} from 'src/components/Text/Text.protractor.driver';
+
+export const helperContentDriverFactory = (element: ElementFinder) => {
+  const title = () => element.$(byDataHook(DataHooks.title));
+  const text = () => element.$(byDataHook(DataHooks.text));
+
+  return {
+    hasTitle: async () => title().isPresent(),
+    getTitleContent: async () => textDriverFactory(title()).getText(),
```

Comment:
Instead of having `hasTitle`, we can just have getTitleContent. But up to you. In any case, this `getTitleContent` should be safer and check that the title exists, otherwise an exception will trigger.